<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaEditor</class>
 <widget class="QMainWindow" name="VentanaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Top_Bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(35, 35, 35);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_toggle">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_top">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Content">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_left_menu">
         <property name="minimumSize">
          <size>
           <width>212</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 46, 109);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="btn_chat">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>100</y>
               <width>171</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	font: 11.5pt &quot;Calibri&quot;;
	color: rgb(255, 255, 255);
	padding-left:-100px;
	background-color: rgb(0, 46, 109);
	border: 0px solid;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(93, 129, 163);
}


	
	
	
	
</string>
             </property>
             <property name="text">
              <string>Chat</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>20</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
font: 11.5pt &quot;Calibri&quot;;</string>
             </property>
             <property name="text">
              <string>Nombre_user</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>31</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
font: 25 11pt &quot;Calibri Light&quot;;
background-color: transparent;</string>
             </property>
             <property name="text">
              <string>Editor</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_cerrarSesion">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>450</y>
               <width>171</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	font: 9pt &quot;Calibri&quot;;
	color: rgb(255, 255, 255);
	padding-left:-50px;
	background-color: rgb(0, 46, 109);
	border: 0px solid;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(93, 129, 163);
}


	
	
	
	
</string>
             </property>
             <property name="text">
              <string>Cerrar Sesi√≥n</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_about">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>180</y>
               <width>171</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	font: 11.5pt &quot;Calibri&quot;;
	color: rgb(255, 255, 255);
	padding-left:-40px;
	background-color: rgb(0, 46, 109);
	border: 0px solid;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(93, 129, 163);
}


	
	
	
	
</string>
             </property>
             <property name="text">
              <string>Sobre Foxy Bot</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>10</y>
               <width>51</width>
               <height>41</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../img/ico.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_menuContenido">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>140</y>
               <width>171</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	font: 11.5pt &quot;Calibri&quot;;
	color: rgb(255, 255, 255);
	padding-left:-10px;
	background-color: rgb(0, 46, 109);
	border: 0px solid;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(93, 129, 163);
}


	
	
	
	
</string>
             </property>
             <property name="text">
              <string>Men√∫ de Contenido</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_pages">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(217, 217, 217);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(217, 217, 217);</string>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page_1">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QFrame" name="frame">
                <property name="styleSheet">
                 <string notr="true">QFrame{
	background-color: rgb(217, 217, 217);
	border-radius:26px;

}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QFrame" name="frame_input_messages">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>410</y>
                   <width>731</width>
                   <height>71</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	
	
	
	padding-top:1px;
	border-radius:25px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QPushButton" name="pushButton">
                  <property name="geometry">
                   <rect>
                    <x>660</x>
                    <y>10</y>
                    <width>51</width>
                    <height>51</height>
                   </rect>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	background-color: rgb(167, 182, 203);
	padding-top:1px;
	border-radius:25px;
	border-color: rgb(0, 46, 109);
}

QPushButton:hover {
	background-color: rgb(93, 129, 163);
}</string>
                  </property>
                  <property name="text">
                   <string>Enviar</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_5">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>0</y>
                    <width>61</width>
                    <height>61</height>
                   </rect>
                  </property>
                  <property name="contextMenuPolicy">
                   <enum>Qt::CustomContextMenu</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../Downloads/icoUser.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                 <widget class="QFrame" name="frame_4">
                  <property name="geometry">
                   <rect>
                    <x>100</x>
                    <y>11</y>
                    <width>541</width>
                    <height>51</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QFrame{
	
	
	
	
	
	
	background-color: rgb(148, 148, 148);
	padding-top:1px;
	border-radius:25px;
	
	border-color: rgb(0, 46, 109);
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <widget class="QLineEdit" name="lineEdit">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>541</width>
                     <height>51</height>
                    </rect>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLineEdit{
	
	
	
	
	background-color: rgb(217, 217, 217);
	padding-top:1px;
	border-radius:25px;
	
	border-color: rgb(0, 46, 109);
}</string>
                   </property>
                   <property name="placeholderText">
                    <string>¬°Escribe aqu√≠ tus dudas!</string>
                   </property>
                  </widget>
                 </widget>
                 <zorder>frame_4</zorder>
                 <zorder>pushButton</zorder>
                 <zorder>label_5</zorder>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="1" column="3">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Soluci√≥n</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="4">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Palabras clave</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Titulo</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="1" column="5">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Fecha de modificacion</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Area de servicios</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="5">
                 <widget class="QPushButton" name="btn_Nuevo">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	
	font: 14pt &quot;Calibri&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 46, 109);
	border: 0px solid;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(93, 129, 163);
}


	
	
	
	
</string>
                  </property>
                  <property name="text">
                   <string>+Nuevo</string>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Contenido: Preguntas/Respuestas de Foxy Bot &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_8">
             <widget class="QFrame" name="frame_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>781</width>
                <height>501</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
	background-color: rgb(217, 217, 217);
	border-radius:26px;

}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>190</y>
                 <width>141</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Calibri&quot;;</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;FOXYBOT v0.1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
